tests/testthat/test-g_mmrm.R

```diff
@@ -24,17 +24,17 @@ fit_mmrm_no_arms_object <- fit_mmrm(
 # g_mmrm_diagnostic ----
 
 test_that("g_mmrm_diagnostic works well with defaults", {
-  result <- expect_silent(g_mmrm_diagnostic(fit_mmrm_object))
+  result <- g_mmrm_diagnostic(fit_mmrm_object)
```

Comment:
why expect_silent is no more needed in this test file? Do modifications produce warnings or plots now? We might want to keep those checks

Reply:
I forgot honestly. basically a message would be ok here, and a warning would anyway cause the test to fail